### PR TITLE
Added example without hanging indentation.

### PR DESCRIPTION
**Pull request by akuhn**
Added an alternative example without hanging indentation since we seem to use both on our codebase.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,19 @@ Airbnb also maintains a [JavaScript Style Guide][airbnb-javascript].
...
 
+    # good
+    def self.create_translation(
+      phrase_id,
+      phrase_key,
+      target_locale,
+      value,
+      user_id,
+      do_xss_check,
+      allow_verification
```
Comment (ljharb) [inline README.md]: wouldn't this need a trailing comma?

Comment (BMorearty) [inline README.md]: No, that's not allowed by Ruby here.

Comment (ljharb) [inline README.md]: ah oops, nvm then

**Comment by venantius** [comment]
I approve of extra examples :)

**Comment by akuhn** [comment]
Getting back to @ljharb comment about trailing comma. I think function calls allow trailing comma, might be good to distinguish this in the style guide. At the moment it just speaks about "function parameters" without distinguishing between function signature and function callsites. I'll make a PR.

**Pull request merged**